src/main/java/io/kestra/plugin/meta/facebook/posts/SchedulePost.java

```diff
@@ -0,0 +1,139 @@
+package io.kestra.plugin.meta.facebook.posts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.JacksonMapper;
+import io.kestra.core.http.HttpRequest;
+import io.kestra.core.http.HttpResponse;
+import io.kestra.core.http.client.HttpClient;
+import io.kestra.plugin.meta.facebook.AbstractFacebookTask;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import lombok.Builder;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@EqualsAndHashCode
+@Schema(
+    title = "Schedule a post on a Facebook Page",
+    description = "Schedule content to be published at a future time on a Facebook Page"
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Schedule a post for tomorrow",
+            full = true,
+            code = """
+                id: facebook_schedule_post
+                namespace: company.team
+
+                tasks:
+                  - id: schedule_post
+                    type: io.kestra.plugin.meta.facebook.posts.SchedulePost
+                    pageId: "{{ secret('FACEBOOK_PAGE_ID') }}"
+                    accessToken: "{{ secret('FACEBOOK_ACCESS_TOKEN') }}"
+                    message: "This post is scheduled for tomorrow!"
+                    scheduledPublishTime: "{{ now() | dateAdd(1, 'DAYS') | date('yyyy-MM-dd HH:mm:ss') }}"
+                """
+        ),
+        @Example(
+            title = "Schedule a post with Unix timestamp",
+            code = """
+                - id: schedule_unix_post
+                  type: io.kestra.plugin.meta.facebook.posts.SchedulePost
+                  pageId: "{{ secret('FACEBOOK_PAGE_ID') }}"
+                  accessToken: "{{ secret('FACEBOOK_ACCESS_TOKEN') }}"
+                  message: "Scheduled post with timestamp"
+                  scheduledPublishTime: "1735689600"
+                  link: "https://example.com"
+                """
+        )
+    }
+)
+public class SchedulePost extends AbstractFacebookTask {
```

Comment:
Here and in the examples please remove the suffix "Post":

```suggestion
public class Schedule extends AbstractFacebookTask {
```